### DISCUSSION TITLE
Update Fail

### DISCUSSION THREAD

**Discussion by kayakbiker** [mytourbook/mytourbook#1363] [General]
I attempted to update from 23.10 to 24.5 and received this error when attempting to start the new version. 

I'm running windows 10 and installed the version without Java, as it should already be on my computer? There was an initial Windows defender and then Norton blockage when installing, but I think I was able to get past those. 
[1718117066011.log](https://github.com/user-attachments/files/15789963/1718117066011.log)

Any help would be appreciated.

**Comment by FJBDev** [comment]
@kayakbiker 

Your log shows that you are using **Java 8** but MT 24.5 requires **Java 17.**

`java.version=1.8.0_351`

The best and easiest solution for you would be to download MT bundled with Java [here](https://github.com/mytourbook/mytourbook/releases/download/24.5.0_2024-05-15_0536/mytourbook-24.5.0-win-setup-with-java-runtime.exe)

I hope this helps!

**Reply by kayakbiker** [reply]
That solved the problem. My bad for not installing with Java. 

Thanks FJBDev!
